openmls/src/group/tests/test_validation.rs

```diff
@@ -213,3 +224,371 @@ fn test_valsem2() {
         ))
     );
 }
+
+// ValSem3 Epoch
+#[test]
+fn test_valsem3() {
+    let ValidationTestSetup {
+        backend,
+        mut alice_group,
+        _alice_credential: _,
+        _bob_credential: _,
+        _alice_key_package: _,
+        bob_key_package,
+    } = validation_test_setup(WireFormat::MlsPlaintext);
+
+    let (message, _welcome) = alice_group
+        .add_members(&backend, &[bob_key_package])
+        .expect("Could not add member.");
+
+    let serialized_message = message
+        .tls_serialize_detached()
+        .expect("Could not serialize message.");
+
+    let mut plaintext = VerifiableMlsPlaintext::tls_deserialize(&mut serialized_message.as_slice())
+        .expect("Could not deserialize message.");
+
+    plaintext.set_epoch(GroupEpoch(100));
```

Comment:
Since the Epoch can be both too high and too low, it would be nice to have one test case for each.